sstables_loader.cc

```diff
@@ -139,25 +140,33 @@ class sstable_streamer {
         });
     }
 
-    future<> stream(bool primary_replica_only);
-private:
+    virtual ~sstable_streamer() {}
```

Comment:
Let's add some words in the commit log for the motivation for the virtualization.

Reply:
Ok.